.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "cryptol-specs"]
 	path = cryptol-specs
 	url = git://github.com/GaloisInc/cryptol-specs
+[submodule "blst_patched"]
```

Comment:
I think this is a reasonable approach to get code from a more recent version, but clearly we will want to limit the number of different versions we deal with.

The name may be a bit confusing, especially in the build scripts I was thrown by the fact that we apply patches to "blst" and not "blst_patched" and was thinking, "but then `blst` is the patched version and `blst_patched` is not!?"  Maybe a name something like "newer_blst" that does not refer to patching  would avoid this.